content/knowledge-base/guides/import-existing-resources.md

```diff
@@ -0,0 +1,262 @@
+
+Crossplane can discover and import existing Provider resources by matching the
+`external-name` annotation in a managed resource. 
+
+To import an existing external resource in a Provider, create a new managed
+resource with a `metadata.annotations.crossplane.io/external-name` value. Set
+the `external-name` to the name of the resource in the Provider.
+
+For example, to import an existing GCP Network named 
+{{<hover label="annotation" line="5">}}my-existing-network{{</hover>}},
+create a new managed resource and use the 
+{{<hover label="annotation" line="5">}}my-existing-network{{</hover>}} in the
+annotation. 
+
+```yaml {label="annotation"}
+apiVersion: compute.gcp.crossplane.io/v1beta1
+kind: Network
+metadata:
+  annotations:
+    crossplane.io/external-name: my-existing-network
+```
+
+The {{<hover label="name" line="5">}}metadata.name{{</hover>}} 
+field can be anything you want. For example, 
+{{<hover label="name" line="5">}}imported-network{{</hover>}}. 
+
+{{< hint "note" >}}
+This name is the 
+name of the Kubernetes object. It's not related to the resource name inside the
+Provider. 
+{{< /hint >}}
+
+```yaml {label="name"}
+apiVersion: compute.gcp.crossplane.io/v1beta1
+kind: Network
+metadata:
+  annotations:
+    name: imported-network
+    crossplane.io/external-name: my-existing-network
+```
+
+Leave the 
+{{<hover label="fp" line="8">}}spec.forProvider{{</hover>}} field empty. 
+Crossplane imports the settings and automatically applies them to the managed 
+resource. 
+
+```yaml {label="fp",copy-lines="all"}
+apiVersion: compute.gcp.crossplane.io/v1beta1
+kind: Network
+metadata:
+  annotations:
+    name: imported-network
+    crossplane.io/external-name: my-existing-network
+spec:
+  forProvider: {}
+```
+
+{{< hint "important" >}}
+If the managed resource has _required_ fields in the 
+{{<hover label="fp" line="8">}}spec.forProvider{{</hover>}} you must add it to
+the `forProvider` field. 
+
+The values of those fields must match what's inside the Provider or Crossplane
+overwrites the existing values.
+{{< /hint >}}
+
+Crossplane now controls and manages this imported resource. Any changes to the
+managed resource `spec` changes the external resource.
+
+## Import resources automatically 
+
+Automatically import external resources with the
+`ObserveOnly` [`managementPolicy`]({{<ref "/v1.12/concepts/managed-resources#managementpolicy">}}).
+
+Crossplane imports `ObserveOnly` resources but never changes or deletes the
+resources.
+
+{{<hint "important" >}}
+The managed resource `managementPolicy` option is an alpha feature. 
+
+Enable the `managementPolicy` in a provider with `--enable-management-policies` 
+in a 
+[ControllerConfig]({{<ref "/v1.12/concepts/providers#controller-configuration" >}}).
+{{< /hint >}}
+
+<!-- vale off -->
+### Apply the ObserveOnly managementPolicy
+<!-- vale on -->
+
+Create a new managed resource with {{<hover label="oo-policy" line="4">}}managementPolicy: ObserveOnly{{</hover>}} set. 
```

Comment:
This was written before your dope suggestion to selectively disable the copy button. I've updated the blocks to disable them. I've added some more explicit language and added the annotation to disable the copy button (this branch doesn't have the feature, but it will be there when it's merged to master)

![Screenshot 2023-06-27 at 11 01 57 AM](https://github.com/crossplane/docs/assets/10537576/ea5db79d-e871-4603-949c-eaafdc271475)